apps/services/user-profile/src/app/v2/userToken.controller.ts

```diff
@@ -0,0 +1,47 @@
+import { ApiSecurity, ApiTags } from '@nestjs/swagger'
+import { Controller, Get, UseGuards, Headers } from '@nestjs/common'
+
+import { IdsAuthGuard, Scopes, ScopesGuard } from '@island.is/auth-nest-tools'
+import { UserProfileScope } from '@island.is/auth/scopes'
+import { Audit } from '@island.is/nest/audit'
+import { Documentation } from '@island.is/nest/swagger'
+
+import { UserTokenService } from './userToken.service'
+import { UserDeviceTokenDto } from '../user-profile/dto/userDeviceToken.dto'
+
+const namespace = '@island.is/user-profile/v2/userTokens'
+
+@UseGuards(IdsAuthGuard, ScopesGuard)
+@Scopes(UserProfileScope.admin)
+@ApiTags('v2/userTokens')
+@ApiSecurity('oauth2', [UserProfileScope.admin])
+@Controller({
+  path: 'userTokens',
```

Comment:
The device tokens are sub-resource of the users so we should let the path be `users/.national-id/device-tokens`

```suggestion
  path: 'users/.national-id/device-tokens',
```